File: SingleAgenda/SingleAgenda.Entities/Location/Address.cs
using SingleAgenda.Entities.Base;
using System;
using System.Collections.Generic;
using System.Text;

namespace SingleAgenda.Entities.Location
{
    public class Address
        : EntityBase
    {

        public string ZipCode { get; set; }
        public string Country { get; set; }
        public string State { get; set; }
        public string City { get; set; }
        public string Description { get; set; }

    }
}

File: web-portal/src/app/core/model/location/address.ts
import { BaseModel } from "../base/base-model";

export class Address extends BaseModel {

    constructor() {
        super();
    }

    public zipCode: string = "";
    public country: string = "";
    public state: string = "";
    public city: string = "";
    public description: string = "";
    public personId: number = 0;

}